python/ray/tests/test_state_api.py

```diff
@@ -2068,6 +2068,96 @@ def ready(self):
     assert "actor_id" in result.output
 
 
+@pytest.mark.skipif(
+    sys.platform == "win32",
+    reason="Lambda test functions could not be pickled on Windows",
```

Comment:
Quick q; does it still work on windows in this case?

Reply:
Good question - the code should work (the unit testing will work). Just not e2e tested easily.